### PR TITLE
chore: Improve workspace selection screen

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Proposed changes

Some small tweaks to workspace selection screen, forget password screen
## Screenshots
image

iPad
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

[MA-84](https://rocketchat.atlassian.net/browse/MA-84)

[MA-84]: https://rocketchat.atlassian.net/browse/MA-84?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/views/ForgotPasswordView.tsx [hunk 1]
```diff
@@ -68,16 +68,17 @@ const ForgotPasswordView = (): React.ReactElement => {
 		<FormContainer testID='forgot-password-view'>
 			<FormContainerInner>
 				<Text style={[sharedStyles.loginTitle, sharedStyles.textBold, { color: colors.fontTitlesLabels }]}>
-					{I18n.t('Forgot_password')}
+					{I18n.t('Reset_password')}
+				</Text>
+				<Text>
```
Comment (diegolmello) [inline app/views/ForgotPasswordView.tsx:73]: This element doesn't have any styles.

File: app/views/NewServerView/index.tsx [hunk 2]
```diff
@@ -371,10 +335,11 @@ class NewServerView extends React.Component<INewServerViewProps, INewServerViewS
 						onPress={this.submit}
 						disabled={!text || connecting}
 						loading={!connectingOpen && connecting}
-						style={[styles.connectButton, { marginTop: 16 }]}
+						style={[styles.connectButton]}
```
Comment (diegolmello) [inline app/views/NewServerView/index.tsx:338]:
```suggestion
						style={styles.connectButton}
```

File: app/views/RegisterView.tsx [hunk 3]
```diff
@@ -99,8 +101,8 @@ class RegisterView extends React.Component<IProps, any> {
 	}
 
 	login = () => {
-		const { navigation, Site_Name } = this.props;
-		navigation.navigate('LoginView', { title: Site_Name });
+		const { navigation, Site_Url } = this.props;
+		navigation.navigate('LoginView', { title: new parse(Site_Url).hostname });
```
Comment (diegolmello) [inline app/views/RegisterView.tsx:105]: You've replaced Site_Name usage, so you need to remove the prop as well.

File: app/views/NewServerView/index.tsx [hunk 4]
```diff
@@ -289,34 +277,29 @@ class NewServerView extends React.Component<INewServerViewProps, INewServerViewS
...
-					testID='new-server-choose-certificate'>
-					<Text style={[styles.chooseCertificate, { color: themes[theme].fontInfo, fontSize: 13 }]}>
-						{certificate ?? I18n.t('Apply_Your_Certificate')}
-					</Text>
-				</TouchableOpacity>
+					testID='new-server-choose-certificate'
+					title={certificate ?? I18n.t('Apply_Certificate')}
+					type='secondary'
+					disabled={connecting}
+					style={{ marginTop: 12, width: '100%' }}
```
Comment (diegolmello) [inline app/views/NewServerView/index.tsx:293]: You should use flexbox instead of width: 100%

File: app/views/RegisterView.tsx (hunk 3 above)
Comment (yash-rajpal) [reply app/views/RegisterView.tsx:104]: Done!

**Comment by yash-rajpal** [comment]
Let me check the breaking E2E tests and update them accordingly.

**Review by diegolmello — CHANGES_REQUESTED** [review]
`Apply certificate` button is quite different from Figma.

### Yours
image

### Figma
image


You could add a `small` prop to `Button`.

File: app/containers/TextInput/FormTextInput.tsx [hunk 5]
```diff
@@ -18,7 +18,6 @@ const styles = StyleSheet.create({
 		marginBottom: 10
 	},
 	label: {
-		marginBottom: 10,
 		fontSize: 14,
```
Comment (diegolmello) [inline app/containers/TextInput/FormTextInput.tsx:21]: Size 16 and line height 22 on Figma, right?

File: app/containers/TextInput/FormTextInput.tsx [hunk 6]
```diff
@@ -29,7 +28,8 @@ const styles = StyleSheet.create({
 		paddingHorizontal: 16,
 		paddingVertical: 10,
 		borderWidth: 1,
-		borderRadius: 2
+		borderRadius: 2,
+		marginTop: 10
```
Comment (diegolmello) [inline app/containers/TextInput/FormTextInput.tsx:32]: gap 4

**Review by diegolmello — CHANGES_REQUESTED** [review]

File: app/containers/TextInput/FormTextInput.tsx (hunk 5 above)
Comment (yash-rajpal) [reply app/containers/TextInput/FormTextInput.tsx:21]: Should we update the styles of this reusable component to match Figma, as this component is also used at many other places? 🤔

Comment (diegolmello) [reply app/containers/TextInput/FormTextInput.tsx:21]: Yes

**Review by diegolmello — APPROVED** [review]

**Pull request merged**